src/adcm_pytest_plugin/fixtures_data/cluster_with_service/config.yaml

```diff
@@ -0,0 +1,37 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Add actions and configs to all levels (cluster, service, component)

It will be nice to see import here also!

Reply:
added